crates/bevy_pbr/src/atmosphere/mod.rs

```diff
@@ -25,6 +25,12 @@
 //! at once is untested, and might not be physically accurate. These may be
 //! integrated into a single module in the future.
 //!
+//! This plugin works in web browsers too! While the visuals might look slightly
+//! different from native platforms, that's because web browsers handle some advanced
+//! graphics features differently. Specifically, when calculating how light travels
+//! through the atmosphere, we use a simpler averaging technique instead of the more
+//! complex blending operations. This difference will be resolved in a future release.
+//!
```

Comment:
```suggestion
//! On web platforms, atmosphere rendering will look slightly different. Specifically, when calculating how light travels
//! through the atmosphere, we use a simpler averaging technique instead of the more
//! complex blending operations. This difference will be resolved for WebGPU in a future release.
//!
```

Imo something like this.